docs/source/author/content_style_guide.rst

```diff
@@ -0,0 +1,231 @@
+Content Style Guide
+##############################################################################
+
+.. note::
+
+  This page is currently intended for internal staff only.
+
+
+.. contents:: Contents
+  :local:
+
+ 
+Writing Style
+==============================================================================
+
+When using pronouns in reference to a hypothetical person, gender neutral pronouns (they/their/them) should be used. 
+ 
+All documents (other than those for internal use only) must be written clearly and simply so that a non-expert is able to understand them.
+Preferably documents should be readable for students. 
+ 
+Any jargon used needs to be clearly explained. 
+
+
+Capitalisation Rules
+------------------------------------------------------------------------------
+
+In the majority of cases capitalisation should not be used for keywords and titles, with the following exceptions, where the phrase refers to a commonly used term that is often capitilised in the literature:
```

Comment:
so I was kind of on the fence with this one, originally it did comply to their capitalisation rules, but then it doesn't match the rest of the docs so I changed it. I think that's fine though because these are rules for the content, not our documentation. Unless you've spotted something I've missed?